hw/ip/usbdev/doc/_index.md

```diff
@@ -125,8 +126,8 @@ The following table summarizes how the different input signals relate to the USB
 
 |  External Pins | Internal Signals | Notes |
 |----------------|------------------|-------|
-| D+, D-         | d_i              | Data input for interfacing with a differential USB transceiver. Used in differential RX mode only. |
-| \"             | dp_i, dn_i       | Single-ended data input signals. These signals are used to detect the SE0 link state in differential RX mode. They can further be used to interface to regular IO cells for prototyping on an FPGA, but such an interface will probably not be USB compliant. |
+| D+, D-         | dp_i, dn_i       | Differential data input signals, intended to go directly to pads for supported targets. These signals are used to detect the SE0 link state, and if a differential receiver is not present, they are also used for K and J symbols. On an FPGA, the components should be used with a USB transceiver, as the bidirectional regular IO cells will likely not be USB compliant. |
```

Comment:
I do wonder how much we should go on about the regular cells being non-compliant, because they seem to work so well and this config is quite popular.